core/src/hashes/blake2b256.rs

```diff
@@ -0,0 +1,78 @@
+use crate::hashes::marked::MarkedDigestWriter;
+use blake2_rfc::blake2b::Blake2b;
+use std::io::{Result as IOResult, Write};
+
+/// A blake2b256 digest.
+pub type Blake2b256Digest = [u8; 32];
+
+/// A struct that exposes a Blake256 `Write` interface.
+///
+/// ```
+/// # use std::io::{Result};
+/// use std::io::Write;
+/// use riemann_core::hashes::{Blake2b256Writer, MarkedDigestWriter};
+///
+/// # fn main() -> Result<()> {
+/// let mut w = Blake2b256Writer::default();
+/// # let data = [0u8; 32];
+///
+/// // Writing more than once will update the hasher.
+/// w.write(&data)?;
+///
+/// // Call finish to consume the produce the hasher and produce the digest.
```

Comment:
propagated my comment typo  from the other file. mind fixing both here and there? :rofl:

Reply:
Pushed a fix to both comments. `consume the produce` lol